### PR TITLE
Actions - Refersion

### LINKED ISSUE #3757: Refersion
### App
Refersion

### Triggers
* New Affiliate

### Actions
* Add Conversion Cancellation
* Create Affiliate
* Get Affiliate
* Send Order to Refersion

### PR DESCRIPTION
**Pull request by vellames**
Closes #3757

It was not possible to implement all actions and sources because the free account was limited in some ways

### CODE REVIEW

File: components/refersion/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/refersion",
-  "version": "0.0.4",
+  "version": "0.0.1",
```
Comment (luancazarine) [inline components/refersion/package.json:3]:
```suggestion
  "version": "0.0.5",
```

File: components/refersion/actions/get-affiliate/get-affiliate.mjs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  key: "refersion-get-affiliate",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    id: {
+      type: "string",
+      label: "Id",
+      description: "The affiliate's ID",
+      optional: true,
```
Comment (luancazarine) [inline components/refersion/actions/get-affiliate/get-affiliate.mjs:16]:
```suggestion
      propDefinition: [
        app,
        "affiliateId",
      ],
```

File: components/refersion/actions/manual-commission-credit/manual-commission-credit.mjs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    },
+  },
+  async run({ $ }) {
+    const data = {
+      id: this.id,
+      commission: this.commission,
+      status: this.status,
+      currency: this.currency,
+    };
+    const res = await this.app.createManualCommissionCredit(data, $);
```
Comment (luancazarine) [inline components/refersion/actions/manual-commission-credit/manual-commission-credit.mjs:49]:
```suggestion
    const {
        app,
        ...data
    } = this;
    
    const res = await app.createManualCommissionCredit(data, $);
```

**Review by luancazarine — CHANGES_REQUESTED** [review]
Hi @vellames 
Thanks for your contribution, I just left some comments, please verify if the those are applicable.

**Review by luancazarine — DISMISSED** [review]
LGTM!

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[refersion_1410.pdf](https://github.com/PipedreamHQ/pipedream/files/11007753/refersion_1410.pdf)

**Review by luancazarine — APPROVED** [review]

**Review by luancazarine — APPROVED** [review]

**Pull request closed without merging**